src/sections/Faq/index.js

```diff
@@ -14,11 +14,7 @@ import {
     CloseIcon
 } from "../../reusecore/Accordion";
 
-import { IoIosArrowDown } from "react-icons/io";
-import { FaPlus } from "react-icons/fa";
-
-import particle1 from "../../assets/images/app/particle/07.png";
-import particle2 from "../../assets/images/app/particle/08.png";
+import { IoIosArrowDown, IoIosArrowUp } from "react-icons/io";
 
 import data from "../../assets/data/faq";
```

Comment:
Assuming that this faq would be different for company from meshery. It should be a separate file.

Reply:
What does this datafile look like? Will each FAQ be written to an mdx (or other type) file?

Reply:
Its an array

Reply:
@Jashpatel1 Change this. And what's the update on search?